src/components/Grid/Grid.jsx

```diff
@@ -103,17 +102,11 @@ const Grid = ({
 
   if (!busy && items?.length === 0) {
     return (
-      <Box {...props}>
-        <Flex
-          height='64px'
-          backgroundColor='white-smoke'
-          alignItems='center'
-          justifyContent='center'>
-          <Typography variant='labelSmall' color='lighterText'>
-            {emptyGridText}
-          </Typography>
-        </Flex>
-      </Box>
+      <Flex height='64px' alignItems='center' justifyContent='center'>
```

Comment:
Why not accepting and propagating other props to the Flex??
```suggestion
      <Flex height='64px' alignItems='center' justifyContent='center' {...props}>
```